roles/geerlingguy.nginx/templates/nginx.conf.j2

```diff
@@ -47,7 +47,32 @@ http {
 {% endblock %}
 
 {% block http_gzip %}
-    # gzip on;
+    gzip on;
+    gzip_proxied any;
+    gzip_static on;
+    gzip_http_version 1.0;
+    gzip_disable "MSIE [1-6]\.";
+    gzip_vary on;
+    gzip_comp_level 6;
+    gzip_types
+        text/plain
+        text/css
+        text/csv
+        text/xml
+        text/javascript
+        application/javascript
+        application/x-javascript
+        application/json
+        application/xml
+        application/xml+rss
+        application/xhtml+xml
+        application/x-font-ttf
+        application/x-font-opentype
+        image/svg+xml
+        image/x-icon;
+    gzip_buffers 16 8k;
+    gzip_min_length 512;
+{% endblock %}
```

Comment:
You've doubled this line!

Reply:
It's the endblock line that appears twice, and you've now removed CSV from the list?

Reply:
uups you're quite right ! should be better now